cfconfiguration/cffinder_unix.go

```diff
@@ -0,0 +1,50 @@
+// +build !windows
+
+package cfconfiguration
+
+import "go/build"
+import "os"
+import "os/user"
+
+var checkFileExistence = func (name string) (os.FileInfo, error) { return os.Stat(name) }
```

Comment:
Anyone better idea for library functions patching for tests?

Reply:
I think a better way would be to create an interface and pass it as an argument to a function - then it will allow for creating a mock implementation for a sake of tests (`ISP` + `DIP` rules from `SOLID`).

Reply:
+1